src/App.tsx

```diff
@@ -4,6 +4,10 @@
 
 import React from 'react';
 import Menu from './components/Menu';
+import { GraphComponent, InfoboxComponent } from './components/BaseComponent';
+import Example from './components/Example';
+// Linting kommer klaga på att Box ligger här, men ligger den högre upp funkar inte webbsidan
```

Comment:
for some reason 
![image](https://user-images.githubusercontent.com/95719901/231196022-ddb45ab6-b3f4-4cbd-b3dc-7403c9f6bf79.png)
this function is undefined when box is imported before Example. Not sure if it is worth investigating further